geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/query/QueryProcess.scala

```diff
@@ -98,6 +102,9 @@ class QueryVisitor(features: SimpleFeatureCollection,
     source.getFeatures(query)
   }
 
+  override def getExpressions: java.util.List[Expression] = {
+    origSft.getAttributeDescriptors.map(ad => ff.property(ad.getLocalName)).toList
```

Comment:
I don't think so. If I understand it right, this is just telling GeoServer what properties we'll use so that it knows the feature collection doesn't need retyped.